README.md

```diff
@@ -10,13 +10,14 @@ entry point for the app. The other files provide the routes.
 Other related Python code that implement features are within `/lib`.
 
 To Do:
-* [x] validate cicd infra (using placeholder app template)
-* [x] validate pr validation
-* [x] create python flask app
-* [ ] add test steps for cicd
-* [x] add build & push-to-ecr steps for cicd
-* [x] create [application cloudformation template](cicd/3-app/aiproxy/template.yml)
-* [ ] authentication
+
+- [x] validate cicd infra (using placeholder app template)
+- [x] validate pr validation
+- [x] create python flask app
+- [ ] add test steps for cicd
+- [x] add build & push-to-ecr steps for cicd
+- [x] create [application cloudformation template](cicd/3-app/aiproxy/template.yml)
+- [ ] authentication
```

Comment:
in the future please try to isolate broad cosmetic changes to a separate commit so that it's easier for reviewers to find the substantial changes.

Reply:
I think these were a result of some auto linting in my VS Code.